src/app/__tests__/__snapshots__/sitemap.test.ts.snap

```diff
@@ -45,11 +45,11 @@ exports[`sitemap > generates the expected sitemap snapshot 1`] = `
     "url": "https://adoptium.net/page-2",
   },
   {
-    "lastModified": 2025-05-01T00:00:00.000Z,
```

Comment:
I think these changes are unintentional (although I also saw them on my side which is weird)

Reply:
I think the update-snapshot did that...